modules/custom/d_p_side_tiles/templates/paragraph--d-p-side-tiles.html.twig

```diff
@@ -65,16 +65,15 @@
   'd-p-side-tiles-content-fields',
 ] %}
 {% block paragraph %}
-
 <section {{ wrapper_attributes }}>
   <div {{ attributes.addClass(classes) }}>
     <div {{ tiles_wrapper_attributes.addClass(tiles_wrapper_classes) }}>
       <div {{ images_side_attributes.addClass(images_side_classes) }}>
-        {{ content.field_d_media_image }}
+        {{ content.group_tiles_content.field_d_media_image }}
```

Comment:
Let's create 3 groups here, identically as in web/profiles/contrib/droopler/modules/custom/d_p_side_image/templates/paragraph--d-p-side-image.html.twig:

`group_d_side_tiles_content` -> text content of paragraph
`group_d_side_tiles_cta` -> cta button
`group_d_side_tiles_gallery` -> the tiles gallery for masonry